causalml/inference/meta/rlearner.py

```diff
@@ -66,21 +67,37 @@ def __repr__(self):
                                          self.model_mu.__repr__(),
                                          self.model_tau.__repr__()))
 
-    def fit(self, X, p, treatment, y, verbose=True):
+    def fit(self, X, treatment, y, p=None, return_p_score=False, verbose=True):
```

Comment:
Instead of returning `p` in `fit()`, why don't we make it available as a member variable and member function e.g. `self.propensity` and `self.get_propensity()`? We can initialize `self.propensity` as `None` and set it in `fit()` with either the input argument `p` if passed or the one computed internally here.

Reply:
Thanks for the suggestion here! Changed accordingly.